adapters/kubient/kubient.go

```diff
@@ -28,6 +28,24 @@ func (adapter *KubientAdapter) MakeRequests(
 	requestsToBidder []*adapters.RequestData,
 	errs []error,
 ) {
+	var bidderExt adapters.ExtImpBidder
+	if err := json.Unmarshal(openRTBRequest.Imp[0].Ext, &bidderExt); err != nil {
```

Comment:
Please do not assume that there is at least 1 imp in the request. It's a common approach to check the number of Imps at the start of this method and exit early:

```
	if len(request.Imp) == 0 {
		return nil, []error{&errortypes.BadInput{
			Message: "No impression in the bid request",
		}}
	}
```

Please also add a test case with no impressions.

Reply:
@SyntaxNode added test for num of imps. Every imp object is checked.